hackathon_site/dashboard/frontend/src/pages/Orders/Orders.tsx

```diff
@@ -69,30 +75,29 @@ const Orders = () => {
                         </Grid>
                         <Grid item lg={12}>
                             <Grid container spacing={1} direction="row">
-                                {Array(12)
-                                    .fill("order card component")
-                                    .map((str, idx) => (
-                                        <Grid
-                                            item
-                                            lg={3}
-                                            md={4}
-                                            sm={6}
-                                            xs={12}
-                                            key={idx}
-                                        >
-                                            <div
-                                                style={{
-                                                    border: "1px solid black",
-                                                    borderRadius: "5px",
-                                                    padding: "10px",
-                                                    textAlign: "center",
-                                                    backgroundColor: "lightblue",
-                                                }}
-                                            >
-                                                {str}
-                                            </div>
-                                        </Grid>
-                                    ))}
+                                {mockPendingOrders.map((order, idx) => (
+                                    <Grid
+                                        item
+                                        lg={3}
+                                        md={4}
+                                        sm={6}
+                                        xs={12}
+                                        key={idx}
+                                        onClick={() => handleClick(order.id)}
+                                    >
+                                        <OrderCard
+                                            teamId={order.team_id}
+                                            orderQuantity={order.items.length}
+                                            timeOrdered={new Date(
+                                                order.created_at
```

Comment:
On the OrderCard could you change the label from **Time ordered --> Time** and change the date-time formay to "Month Day, HH:MM" in 24h time e.g. Feb 21, 13:05
Only because orders are going to go for 2 weeks so time is not enough info, we need date too